client/reader/user-profile/components/user-profile-header/index.tsx

```diff
@@ -78,10 +78,13 @@ const UserProfileHeader = ( { user }: UserProfileHeaderProps ): JSX.Element => {
 									{ user.bio }
 								</span>
 								{ isClamped && user.profile_URL && (
-									<a className="user-profile-header__bio-desc-link" href={ user.profile_URL }>
-										{ translate( 'Read More' ) }{ ' ' }
-										<Icon width={ 18 } height={ 18 } icon={ external } />
-									</a>
+									<>
+										<div className="user-profile-header__bio-desc-fader"></div>
+										<a className="user-profile-header__bio-desc-link" href={ user.profile_URL }>
```

Comment:
Should we open the link in a new tab?

Reply:
Great point, I think yes!

1. its an out of app flow, so we should make it open externally
2. the icon we are showing i think generally represents external links that should be opened in a new tab.

Reply:
Fixed 👍